src/main/java/cloud/fogbow/common/util/PropertiesUtil.java

```diff
@@ -15,23 +15,10 @@
     private static final Logger LOGGER = Logger.getLogger(PropertiesUtil.class.getName());
 
     public static Properties readProperties(String configFileName) throws FatalErrorException {
-        List<String> configFilesNames = new ArrayList<>();
-        configFilesNames.add(configFileName);
-        return readProperties(configFilesNames);
+        return loadProperties(configFileName);
     }
 
-    public static Properties readProperties(List<String> configFilesNames) throws FatalErrorException {
-        Properties properties = new Properties();
-
-        for (String fileName : configFilesNames) {
-            Properties mProperties = loadProperties(fileName);
-            properties.putAll(mProperties);
-        }
-
-        return properties;
-    }
-
-    private static Properties loadProperties(String fileName) throws FatalErrorException {
+    protected static Properties loadProperties(String fileName) throws FatalErrorException {
```

Comment:
cool, @ANKerD the code now is simpler :) thanks
so, your modification raised a new issue: why do we need two methods? it seems keeping only the loadProperties would be just fine.

Reply:
Indeed, this code structure was used to support the two readProperties methods. I'll combine the two methods left into one.